app/components/candidate_interface/application_status_tag_component.html.erb

```diff
@@ -6,6 +6,34 @@
   </p>
 <% elsif  @application_choice.offer_deferred? %>
   <p class="govuk-body govuk-body govuk-!-margin-top-2">
-    Your training will now start in <%= (@application_choice.course.start_date + 1.year).strftime('%B %Y') %>.
+  Your training will now start in <%= (@application_choice.course.start_date + 1.year).to_s(:month_and_year) %>.
+  </p>
+<% elsif @application_choice.pending_conditions? || @application_choice.recruited? || @application_choice.offer? %>
+  <p class="govuk-body govuk-body govuk-!-margin-top-2">
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          What to do if you’re unable to start training this year
+        </span>
+      </summary>
+
+      <div class="govuk-details__text govuk-!-padding-bottom-0">
+        Some providers allow you to defer your offer. This means that you could start your course a year later.
+      </div>
+
+      <div class="govuk-details__text govuk-!-padding-bottom-0">
+        Every provider is different, so it may or may not be possible to do this.
+      </div>
+
+      <div class="govuk-details__text govuk-!-padding-bottom-0">
+        Find out by contacting <%= @application_choice.provider.name %>. Asking if it’s possible to defer will not affect your existing offer.
```

Comment:
In the card it looks like these two sentences are meant to be split into two paragraphs.

Reply:
Same as the above